ext/skeleton/.gitignore.in

```diff
@@ -8,11 +9,13 @@ build
 config.guess
 config.h
 config.h.in
+*~
 config.log
 config.nice
 config.status
 config.sub
 configure
+configure~
```

Comment:
not needed after `*~`

Reply:
Thank you! I was on mobile and missed it.